src/components/material/ReviewHearts.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import HeartIconFilled from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/basic/icon-heart-filled.svg";
+import HeartIconGrey from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/basic/icon-heart-grey.svg";
+
+export interface ReviewHeartsProps {
+  amountOfHearts: string;
+}
+
+const ReviewHearts: React.FC<ReviewHeartsProps> = ({ amountOfHearts }) => {
+  const heartArray = amountOfHearts.split("/");
+  const filledHeartsArray = Array.from(
+    { length: Number(heartArray[0]) },
+    (_, i) => i + 1
+  );
+  const emptyHeartsArray = Array.from(
+    { length: Number(heartArray[1]) - Number(heartArray[0]) },
+    (_, i) => i + 1
+  );
+  return (
+    <div
+      className="mb-4"
+      role="figure"
+      aria-label="Rating of this item is 5 out of 6"
```

Comment:
Please make this text translatable and vary according to `filledHeartsArray.length` and `heartArray.length`.